### ISSUE TITLE
Do the techniques need to update if not shown alternative

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#423]
Currently the techniques addresses the case where This key information can be hidden if metadata is missing.

However, now that the five sections were updated to include the alternative string with its ID,  do the techniques need to be updated?

**Comment by GeorgeKerscher** [comment]
We have addressed this problem.
